src/profilGestion.java

```diff
@@ -113,7 +108,7 @@ public static void chargeProfil()
 		{			
 			String str = "";
 			Fichier fi = new Fichier();
-			fi.ouvrir(name + ".sav", "L", true);
```

Comment:
Nope it's a serious error here. Sorry but if true is not there, java will create a new file instead of the old one. Sorry i can maybe easly delete the old file but it will take a lot more time to correct the new bugs (more it's a bug which i have already corrected). Please take note of my multiples comments and adjust your shot. Thanks